packages/contract-addresses/src/index.ts

```diff
@@ -162,6 +43,8 @@ const networkToAddresses: { [networkId: number]: ContractAddresses } = {
  * given networkId.
  */
 export function getContractAddressesForNetworkOrThrow(networkId: NetworkId): ContractAddresses {
+    const networkToAddresses: { [networkId: number]: ContractAddresses } = addresses;
```

Comment:
I guess it could be, if I were adhering strictly to the style in which it was written, but I took the liberty of changing the style while I was in here.  Personally I think it should be a local variable because it's neither exported nor re-used anywhere else.